activerecord/lib/active_record/connection_adapters/abstract/schema_definitions.rb

```diff
@@ -303,7 +303,7 @@ def [](name)
       #   end
       def column(name, type, options = {})
         name = name.to_s
-        type = type.to_sym
+        type = type ? type.to_sym : :__placeholder
```

Comment:
if `type` information is not there should not we just not call `to_sym` on it and pass `nil` down in the stream?